package.json

```diff
@@ -96,7 +96,7 @@
     "is-ci": "^3.0.1",
     "jsdom": "^21.1.0",
     "lint-staged": "^13.1.2",
-    "msw": "^1.1.0",
+    "msw": "1.0.1",
```

Comment:
Had to lock the version because msw was throwing a `ReferenceError`
![mswjs-bug-1-1-0](https://user-images.githubusercontent.com/32243459/225155650-f43a414f-0e26-4081-ac57-f8362718e9e8.png)